common/changes/@microsoft/web-library-build/pgonzal-ae-reads-dts_2018-01-05-22-42.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "packageName": "@microsoft/web-library-build",
+      "type": "none"
```

Comment:
>none [](start = 15, length = 4)

@qz2017 if I already added a "minor" changelog in this PR, and then I run "rush change" a second time, why is it creating this empty file?

Reply:
Did you click enter/enter w/o looking at the options?